### PR TITLE
(SERVER-545) 3x->4x compatibility acceptance tests

### PR DESCRIPTION
**Pull request by nwolfe**
This commit adds a suite of acceptance tests using the nwolfe-simmons
module to exercise various parts of the master/server HTTP API.

----

module: https://github.com/nwolfe/puppet-simmons/tree/0.3.1

### CODE REVIEW

File: dev/scripts/server545_puppet3_compatibility.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 # Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
-export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.05.14T1222'
```
Comment (KevinCorcoran) [inline dev/scripts/server545_puppet3_compatibility.sh]: This comment is directly about the changes in this PR, but it seems a bit weird to me that we have a hard-coded version here ... shouldn't the beaker use the version of the package was built earlier in the pipeline?

Or ... I see this is under `dev/scripts` ... I guess it's not part of the acceptance test suite?

Comment (KevinCorcoran) [inline dev/scripts/server545_puppet3_compatibility.sh]: I should drink more :coffee: before attempting to review beaker-releated PRs...

Comment (nwolfe) [inline dev/scripts/server545_puppet3_compatibility.sh]: Ha yeah this script isn't used by CI at all, not sure exactly why it ended up here in the first place but we can probably delete it once we prove out the CI pipeline.

File: acceptance/lib/puppetserver/acceptance/compat_utils.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+def nonmaster_agents()
+  agents.reject { |agent| agent == master }
+end
+
+def apply_simmons_class(agent, master, studio, classname)
```
Comment (dankreek) [inline acceptance/lib/puppetserver/acceptance/compat_utils.rb]: Simmons class:

Comment (KevinCorcoran) [inline acceptance/lib/puppetserver/acceptance/compat_utils.rb]: :clap:

**Comment by camlow325** [comment]
:+1:

**Pull request merged**